src/RRTMGP.jl

```diff
@@ -11,6 +8,7 @@ import ClimaComms
 include("Parameters.jl")
 import .Parameters as RP
 
+include("artifact_paths.jl")
```

Comment:
A common Julia convention is to name files that contain modules after the module
```suggestion
include("ArtifactPaths.jl")
```